Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -85,10 +85,12 @@ void function InitInGameMPMenu()
 	var gameHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_GAME" )
 	var leaveButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#LEAVE_MATCH" )
 	Hud_AddEventHandler( leaveButton, UIE_CLICK, OnLeaveButton_Activate )
-	#if DEV
-		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Dev" )
-		Hud_AddEventHandler( devButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "DevMenu" ) ) )
-	#endif
```

Comment:
Why remove this?

Reply:
Forgot to restore after i started experimenting with the button, in the early attempts.